assets/src/edit-story/components/canvas/pagenav/index.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { PAGE_NAV_BUTTON_WIDTH, PAGE_NAV_PADDING } from '../../../constants';
+import { useStory } from '../../../app';
+import { LeftArrow, RightArrow } from '../../button';
+
+const Wrapper = styled.div`
+	display: flex;
+	align-items: center;
+	justify-content: flex-start;
+	height: 100%;
+	color:  ${ ( { theme } ) => theme.colors.fg.v1 };
+`;
+
+const Space = styled.div`
+	flex: 0 1 ${ ( { isNext } ) => isNext ? PAGE_NAV_PADDING : 0 }px;
+`;
+
+function PageNav( { isNext } ) {
+	const { state: { pages, currentPageIndex }, actions: { setCurrentPage } } = useStory();
+	const handleClick = useCallback( () => {
+		const newPage = isNext ? pages[ currentPageIndex + 1 ] : pages[ currentPageIndex - 1 ];
+		if ( newPage ) {
+			setCurrentPage( { pageId: newPage.id } );
+		}
+	}, [ setCurrentPage, currentPageIndex, isNext, pages ] );
+	const displayNav = ( isNext && currentPageIndex < pages.length - 1 ) || ( ! isNext && currentPageIndex > 0 );
+	const buttonProps = {
+		isDisabled: ! displayNav,
+		isHidden: ! displayNav,
+		'aria-label': isNext ? 'Next Page' : 'Previous Page',
+		onClick: handleClick,
+		width: PAGE_NAV_BUTTON_WIDTH,
+		height: 40,
+	};
+	return (
+		<Wrapper>
+			<Space isNext={ isNext } />
```

Comment:
Instead of dealing with the spacing here to get the desired layout, this might be more appropriate for the CSS grid definition in `canvasLayout`. But not a huge deal I suppose.

Reply:
Hmm, perhaps yes. The left navigation actually already doesn't have spacing here anyway and is using the grid for that.

Reply:
Changed in 58e9b9c. I wonder if the grid isn't getting confusing this way? Thoughts?

Reply:
We can always improve it from there, but at least the spacing is now in one place. Perhaps #4120 helps with the grid definition already, otherwise we can improve readability of the grid CSS in a separate PR.